tests/run/worktree/test.sh

```diff
@@ -20,5 +20,14 @@ rlJournalStart
             rlRun "tmt run -ar provision -h $method report -vvv"
             rlRun "popd"
         rlPhaseEnd
+
+        rlPhaseStartTest "Ansible ($method) - check verbose attribute"
+            rlRun "pushd data/ansible"
+            rlRun "tmt run -rddd discover provision -h $method prepare \
+                | grep \"ansible-playbook\"\
+                | tee output"
+            rlAssertGrep "-vvv" output
+            rlRun "popd"
+        rlPhaseEnd
```

Comment:
Why have you extended the `/tests/run/worktree` test? Wouldn't `/tests/prepare/ansible` be more suitable for a new ansible prepare feature?